.github/workflows/autoformat.js

```diff
@@ -13,14 +13,18 @@ const createCommitStatus = async (context, github, sha, state) => {
   });
 };
 
-const createStatus = async (context, github) => {
+const createStatus = async (context, github, core) => {
   const { owner, repo } = context.repo;
   const pull_number = context.issue.number;
   const pr = await github.pulls.get({ owner, repo, pull_number });
   const { sha, ref } = pr.data.head;
+  const repository = pr.data.repo.full_name;
   await createCommitStatus(context, github, sha, 'pending');
+  if (repository === 'mlflow/mlflow' && ref === 'master') {
+    core.setFailed('Running autoformat bot against master branch of mlflow/mlflow is not allowed.');
```

Comment:
Added validation to prevent pushing commits to the master branch of `mlflow/mlflow`.